flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/DefaultLeaderElectionServiceTest.java

```diff
@@ -666,66 +667,38 @@ void testErrorIsIgnoredAfterLeaderElectionBeingClosed() throws Exception {
 
                             leaderElection.close();
 
-                            testingLeaderElectionDriver.onFatalError(testException);
+                            testingLeaderElectionDriver.triggerErrorHandling(testException);
                             assertThat(testingContender.getError()).isNull();
                         });
             }
         };
     }
 
-    /**
-     * Tests that we can shut down the DefaultLeaderElectionService if the used LeaderElectionDriver
-     * holds an internal lock. See FLINK-20008 for more details.
-     */
-    @Test
-    void testServiceShutDownWithSynchronizedDriver() throws Exception {
```

Comment:
was this a duplicate?

Reply:
Yes, see the corresponding commit message:
```
[test] Removes testServiceShutDownWithSynchronizedDriver

It should essentially test the same thing as testCloseGrantDeadlock according to the description of FLINK-20008
```